src/components/MoveBackButton.jsx

```diff
@@ -0,0 +1,18 @@
+import {useRouter} from 'next/router';
+
+import BackArrow from './Icons/BackArrow';
+import StyledButton from './StyledButtons';
+
+export default function MoveBackButton() {
+	const router = useRouter();
+	return (
+		<StyledButton
+			onClick={() => {
+				router.push('/');
+			}}
+			variant="goback"
+		>
+			<BackArrow />
+		</StyledButton>
```

Comment:
If I remember correctly you only go back to the home page. If that's the case you could also simply use a `Link` component to achieve the same.

Reply:
in future, to have less files and code, this is the more efficient way, i understand what you mean and i tried it and it works as well but i would have more code at the end.